tests/Resources/Page/ODH/JupyterHub/AccessGroups.resource

```diff
@@ -0,0 +1,161 @@
+    Handle Bad Gateway Page
+    Run Keyword And Continue On Failure   Login Verify Access Level  ${TEST_USER_2.USERNAME}
+    ...                                   ${TEST_USER_2.PASSWORD}    ${TEST_USER_2.AUTH_TYPE}    admin
+    Capture Page Screenshot    perm_admin_std.png
+    Logout Via Button
+    Go To RHODS Dashboard
+    Launch JupyterHub From RHODS Dashboard Link
+    Run Keyword And Continue On Failure   Login Verify Access Level    ${TEST_USER_4.USERNAME}
+    ...                                   ${TEST_USER_4.PASSWORD}    ${TEST_USER_4.AUTH_TYPE}    user
+    Capture Page Screenshot    perm_user_std.png
+    Logout Via Button
+
+Apply New Groups Config Map
+    [Documentation]    Changes the rhods-groups config map to set the new access configuration
+    OpenShiftCLI.Patch    kind=ConfigMap
+    ...                   src={"data":{"admin_groups": "${CUSTOM_ADMINS_GROUP}","allowed_groups": "${CUSTOM_USERS_GROUP}"}}
+    ...                   name=rhods-groups-config   namespace=redhat-ods-applications  type=merge
+    OpenShiftCLI.Patch    kind=ConfigMap
+    ...                   src={"metadata":{"labels": {"opendatahub.io/modified": "true"}}}
+    ...                   name=rhods-groups-config   namespace=redhat-ods-applications  type=merge
+
+Restore RHODS Standard Groups Config Map
+    [Documentation]    Restores the standard rhods-groups config map
+    OpenShiftCLI.Patch    kind=ConfigMap
+    ...                   src={"data":{"admin_groups": "${STANDARD_ADMINS_GROUP}","allowed_groups": "${STANDARD_USERS_GROUP}"}}
+    ...                   name=rhods-groups-config   namespace=redhat-ods-applications  type=merge
+    OpenShiftCLI.Patch    kind=ConfigMap
+    ...                   src={"metadata":{"labels": {"opendatahub.io/modified": "${STANDARD_GROUPS_MODIFIED}"}}}
+    ...                   name=rhods-groups-config   namespace=redhat-ods-applications  type=merge
+
+Restore Standard Configuration
+    [Documentation]    Restores the standard RHODS access configuration
+    Restore RHODS Standard Groups Config Map
+    Rollout JupyterHub
+    Go To    ${OCP_CONSOLE_URL}
+    Add Test Users Back To RHODS Standard Groups
+    Remove Test Users From Custom Groups
+    Delete Custom Groups
+    Check Standard Access Configuration Works As Expected
+
+Set Standard RHODS Groups Variables
+    [Documentation]     Sets the RHODS groups name based on RHODS version
+    ${version_check}=    Is RHODS Version Greater Or Equal Than    1.8.0
+    IF    ${version_check} == True
+        Set Suite Variable    ${STANDARD_ADMINS_GROUP}      dedicated-admins
+        Set Suite Variable    ${STANDARD_USERS_GROUP}       system:authenticated
+        Set Suite Variable    ${STANDARD_GROUPS_MODIFIED}       true
+    ELSE
+        Set Suite Variable    ${STANDARD_ADMINS_GROUP}      rhods-admins
+        Set Suite Variable    ${STANDARD_USERS_GROUP}       rhods-users
+        Set Suite Variable    ${STANDARD_GROUPS_MODIFIED}       false
+    END
```

Comment:
Need to wait for @pablofelix to update install script and figure out what are the standard groups going forward

Reply:
I think now we can unblock this PR